sputnikdao2/src/proposals.rs

```diff
@@ -91,6 +91,8 @@ pub enum ProposalKind {
     },
     /// Just a signaling vote, with no execution.
     Vote,
+    /// Update part of the policy.
```

Comment:
As stated on `AddMemberToRole` and `RemoveMemberFromRole`, can you add `This is short cut to updating the whole policy.` to the comment?